packages/viem/docs/chains/variables/mainnetChains.md

```diff
@@ -6,8 +6,8 @@
 
 # mainnetChains
 
-> `const` **mainnetChains**: (`object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object`)[]
+> `const` **mainnetChains**: (`object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object` \| `object`)[]
```

Comment:
ah nice, i was curious where we kept our `object`s. 
![Image](https://github.com/user-attachments/assets/7993848f-2d58-47dc-b0c1-70f589557ce3)

Reply:
lol looks like we should figure out how to fix this so it actually prints something useful. created this ticket: https://github.com/ethereum-optimism/ecosystem/issues/968